lib/ansible/modules/system/firewalld.py

```diff
@@ -465,6 +501,96 @@ def set_disabled_permanent(self, service, timeout):
         self.update_fw_settings(fw_zone, fw_settings)
 
 
+class DirectRuleTransaction(FirewallTransaction):
```

Comment:
This looks good, would just like other comments addressed so I can understand motivation behind the proposed changes.

Reply:
Sorry, it was silly mistake. I used old file version.

Reply:
No worries, happens sometimes. :+1: